src/bindings/c/tests/ov_test.hpp

```diff
@@ -41,10 +41,11 @@ inline static std::vector<uint8_t> content_from_file(const char* filename, bool
         std::ifstream is(filename, is_binary ? std::ifstream::binary | std::ifstream::in : std::ifstream::in);
         if (is) {
             is.seekg(0, std::ifstream::end);
-            result.resize(is.tellg());
-            if (result.size() > 0) {
+            size_t file_len = is.tellg();
+            result.resize(file_len + 1);
+            if (file_len > 0) {
```

Comment:
Why not?
```suggestion
            if (file_len > 0) {
                result.resize(file_len);
```

Reply:
In order to avoid invalid read size of 1 in memory, we need to add "\0" at the end of result var.